cbor.opam

```diff
@@ -1,20 +1,41 @@
+# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-maintainer: "ygrek@autistici.org"
-authors: "ygrek"
+synopsis: "CBOR encoder/decoder (RFC 7049) - native OCaml implementation"
+description: """
+
+The Concise Binary Object Representation (CBOR) is a data format whose design
+goals include the possibility of extremely small code size, fairly small
+message size, and extensibility without the need for version negotiation.
+"""
+maintainer: ["ygrek@autistici.org"]
+authors: ["ygrek"]
+license: "MIT"
+tags: ["org:ygrek"]
 homepage: "https://github.com/ygrek/ocaml-cbor"
+doc: "https://github.com/ygrek/ocaml-cbor"
 bug-reports: "https://github.com/ygrek/ocaml-cbor/issues"
-dev-repo: "git://github.com/ygrek/ocaml-cbor.git"
-tags: "org:ygrek"
-build: ["dune" "build" "-p" name "-j" jobs]
 depends: [
-  "ocaml" {>= "4.00.0"}
-  "dune" {>= "1.1"}
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.02"}
   "base-bytes"
   "ocplib-endian" {>= "0.6"}
-  "yojson" {with-test & >= "1.6" }
+  "yojson" {with-test & >= "1.6"}
+  "odoc" {with-doc}
 ]
-synopsis: "CBOR encoder/decoder (RFC 7049) - native OCaml implementation"
-description: """
-The Concise Binary Object Representation (CBOR) is a data format whose design goals
-include the possibility of extremely small code size, fairly small message size, and
-extensibility without the need for version negotiation."""
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
```

Comment:
I didn't see this one before, any particular reason? or is it new best practice? I guess it is related to the subsequent `dune install`?